src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="POST"
+      class="form"
+      data-qa="big"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="search"
+          class="icon icon--big"
+        >
+        <input
+          type="text"
+          class="search-bar search-bar--big"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
```

Comment:
closing brace should be at the same level as opening one - this way your layout is better structured and it is easier to distinguish where ends one tag and the other starts
```suggestion
        <input
          type="text"
          class="search-bar search-bar--big"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```